### DISCUSSION TITLE
"cmd+i" conflict with default "Trigger Suggest" shortcut in github co-pilot chat

### DISCUSSION THREAD

**Discussion by Norfeldt** [community/community#58354] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I have to re-map the github co-pilot **"Interactive Editor: Code Chat"** to `"cmd+u"` and **"Chat: Ask Qucik Question"** to `"shift+cmd+u"` to avoid conflicts with **"Trigger Suggest"**. 

I really use trigger suggest a LOT, so wanna give the feedback to reconsider the shortcuts for the new co-pilot chat.

**Comment by reezayn** [comment]
I need answers for this as well
